.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: ruby
+rvm:
+  # SLES15 sp0
+  - 2.4.1
+
+  # Future versions
+  - ruby-head
+script:
+  - 'bundle exec rspec spec'
+  - 'bundle exec rubocop'
```

Comment:
It seems like Travis runs rubocop even if rspec failed: https://travis-ci.org/SUSE/rmt/jobs/258048380